lib/localparticipant.js

```diff
@@ -208,15 +208,29 @@ class LocalParticipant extends Participant {
     this.on('trackEnabled', localTrackEnabled);
     this.on('trackStopped', localTrackStopped);
 
+    const self = this;
```

Comment:
do you need `self` here? Given that you are accessing it only from arrow functions can you not use `this`

Reply:
ah I see that `self` is used by  `stateChanged` below. Maybe convert `stateChanged` also into arrow function?